sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/ItemList.kt

```diff
@@ -14,12 +14,12 @@ fun ItemList(
     onItemClick: (UiModel) -> Unit
 ) {
     LazyColumn {
-        itemsIndexed(items = uiModels) { index, uiModel ->
+        itemsIndexed(items = uiModels) { _, uiModel ->
```

Comment:
Shall we just use `items` instead since we don't need the index anymore? 😄 
```suggestion
        items(uiModels) { uiModel ->
```

Reply:
Thank you 😄 Updated